### PR TITLE
Add support for multiple audio tracks

### PR DESCRIPTION
**Pull request by hicom150**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[x] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Enabling Chromium `experimentalFeatures` I made an attempt to add support for changing between multiple audio tracks.
 
**Is there anything you'd like reviewers to focus on?**
As it need to enable `experimentalFeatures`, check that all other features work as expected 😅

### CODE REVIEW

**Comment by feross** [comment]
Wow, this is a great! I haven't had a chance to look at the code or test it yet (hoping others can help review it in the meantime) but this is exciting!

**Comment by Borewit** [comment]
@hicom now you are a member of the [webtorrent organization](https://github.com/orgs/webtorrent/people), you may want to create branches for PR's directly on the webtorrent repository. That makes "jumping" and operations between branches in my opinion a bit easier.

File: src/renderer/pages/player-page.js [hunk 1]
```diff
@@ -56,6 +54,14 @@ function renderMedia (state) {
...
     } else if (!state.playing.isPaused && mediaElement.paused) {
       mediaElement.play()
+        .then(() => {
+          dispatch('mediaSuccess')
+        })
+        .catch((err) => {
+          if (err.name === 'NotSupportedError') {
+            dispatch('mediaError', 'Codec unsupported')
+          }
+        })
```
[the remark this answers is not in the document]
Comment (Borewit) [reply src/renderer/pages/player-page.js]: I removed these changes using a force push on your branch

File: src/renderer/pages/player-page.js [hunk 2]
```diff
@@ -168,20 +193,6 @@ function renderMedia (state) {
...
-    if (state.playing.type === 'video' &&
-      elem.webkitVideoDecodedByteCount === 0) {
-      dispatch('mediaError', 'Video codec unsupported')
-    } else if (elem.webkitAudioDecodedByteCount === 0) {
-      dispatch('mediaError', 'Audio codec unsupported')
-    } else {
-      dispatch('mediaSuccess')
-      elem.play()
-    }
-  }
```
[the remark this answers is not in the document]
Comment (Borewit) [reply src/renderer/pages/player-page.js]: I removed these changes using a force push on your branch

**Comment by Borewit** [comment]
@hicom150, please do an [amend commit](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) (maybe twice, one time with a dummy change, the second time you do undo the change). Once you have done that, you can [force push](https://git-scm.com/docs/git-push) the branch which will put back your changes from @Borewit  to @hicom150,.

File: src/renderer/pages/player-page.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (hicom150) [reply src/renderer/pages/player-page.js]: Great! Thank you! 👍

**Comment by hicom150** [comment]
I think that this PR needs #1711 to be merged to work correctly in all edge cases, so I will wait just a bit 😅

**Comment by Borewit** [comment]
> I will wait just a bit

Makes sense, let's get #1711 in

**Comment by hicom150** [comment]
Given that #1711 is already merged, I'm going to merge also this one 😉
Thank you for your help @Borewit 💪

**Comment by Borewit** [comment]
Very nice @hicom150, I am glad these improvements are in.

**Pull request merged**